src/Components/AllArticles.jsx

```diff
@@ -9,27 +9,25 @@ const AllArticles = () => {
   useEffect(() => {
     getArticlesFromApi()
       .then((articlesApi) => {
-        console.log(articlesApi, "API");
         setArticles(articlesApi);
         setIsLoading(false);
       })
       .catch((err) => {
-        console.log(err.response);
         setErr(`Not found`);
       });
-  });
+  }, []);
   if (err) return <p>{err}</p>;
   if (isLoading) return <h1> loading</h1>;
-  console.log(articles, "this is data");
   return (
-    <main className="article_section">
+    <main>
       <h2>Articles</h2>
       <ul>
-        {articles.map((data) => {
+        {articles.map((article) => {
           return (
-            <li key={articles.article_id} className="article__list">
-              Topic: {data.topic} {data.title} {data.body}
-              {data.votes} {data.author} {data.comment_count}
+            <li key={article.article_id}>
+              <Link to={`/articles/${article.topic}`}>
+                Topic: {article.topic}. "{article.title}"
+              </Link>
```

Comment:
This is generating a list which functionally works as a list of links to the articles of a specific topic but at a glance it looks the links would take you to the specific article page. It looks like you have some work in the navbar to select a topic which could be a more suitable place to have the link to a specific topic's articles.